src/pushex.ts

```diff
@@ -42,7 +41,7 @@ export class Pushex {
   }
 
   public disconnect() {
-    return new Promise((resolve) => {
+    return new Promise<void>((resolve) => {
```

Comment:
Doesn't hurt older versions of TypeScript, but is required for more recent versions when `resolve` is called with 0 arguments so went ahead and changed it.